student/src/main/java/tds/student/services/PrintService.java

```diff
@@ -248,16 +278,30 @@ public boolean printItemBraille (TestOpportunity testOpp, ItemResponse responseT
         _tdsLogger.rendererWarn (error, "printItemBraille");
         return false;
       }
+
+      ITSAttachment mainBrailleAttachment = brailleAttachments.get(0);
+
+      // the subType in the Item Metadata attachmentList is the only way to determine if there is a Braille Transcript file
+      //  it will follow the convention of being named with an _transcript suffix according to the specification
+      boolean isTranscript = brailleAttachments.size() == 2 && brailleAttachments.get(1).getSubType().endsWith("_transcript");
+
+
       final String requestType = "EMBOSSITEM";
-      String requestValue = brailleAttachment.getFile ();
-      String requestParameters = "FileFormat:"
-          + brailleAttachment.getType ().toUpperCase ();
-      String requestDesc = getItemLabel (responseToPrint);
-      requestDesc = String.format ("%1$s (%2$s)", requestDesc,
-          brailleAttachment.getType ());
-      _oppRepository.submitRequest (testOpp.getOppInstance (),
-          responseToPrint.getPage (), responseToPrint.getPosition (),
-          requestType, requestDesc, requestValue.replace ("\\", "\\\\"), requestParameters);
+      String requestValue = mainBrailleAttachment.getFile();
```

Comment:
it is almost but slightly different.  i was trying to keep a smaller-ish footprint since this is editing existing code